src/unraphael/dash/pages/5_cluster.py

```diff
@@ -0,0 +1,298 @@
+    sharpness. The mean brightness, contrast and sharpness of the images are computed"""
+    
+    col1, col2 = st.columns(2)
+
+    with col1:
+        col1.subheader('Equalization parameters')
+
+        brightness = st.checkbox('Equalize brightness', value=False)
+        contrast   = st.checkbox('Equalize contrast',   value=False)
+        sharpness  = st.checkbox('Equalize sharpness',  value=False)
+
+        preprocess_options = {
+            'brightness': brightness,
+            'contrast': contrast,
+            'sharpness': sharpness,        
+        }
+        
+        all_images = [img for name, img in images.items()]
+                
+        equalized_images = equalize_images(all_images, **preprocess_options)
+    
+    with col2:
+        metrics = compute_metrics(equalized_images)
+        
+        col2.subheader('Metrics after equalization')
+
+        col3, col4 = st.columns((2))
+        col3.metric('Mean Normalized Brightness', metrics['mean_normalized_brightness'])
+        col4.metric('SD Normalized Brightness', metrics['sd_normalized_brightness'])
+        
+        col3.metric('Mean Normalized Contrast', metrics['mean_normalized_contrast'])
+        col4.metric('SD Normalized Contrast', metrics['sd_normalized_contrast'])
+        
+        col3.metric('Mean Normalized Sharpness', metrics['mean_normalized_sharpness'])
+        col4.metric('SD Normalized Sharpness', metrics['sd_normalized_sharpness'])    
+    
+    # Map the equalized images back to their original names
+    return {name: equalized_images[i] for i, name in enumerate(images.keys())}
+    
+
+def align_to_mean_image_widget(*, images: dict[str, np.ndarray]) -> dict[str, np.ndarray]:
+    """This widget aligns all images, preferably to their mean value but other 
+    aligning options are also available. The aligned images are input for thr later
+    clustering of the images"""
+    
+    st.subheader('Alignment parameters')
+    
+    motion_model = st.selectbox(
+            'Motion model:',
+            ['translation',
+             'rigid body',
+             'scaled rotation',
+             'affine',
+             'bilinear'],
+            help = (
+                'The motion model defines the specific transformation one wants to apply.'          
+            ),
+        )
+    
+    feature_method = st.selectbox(
+            'Register and transform a stack of images:',
+            ['to first image',
+             'to mean image',
+             'each image to the previous (already registered) one'],
+            help = (
+                'For more help, see https://pystackreg.readthedocs.io/en/latest/readme.html'          
+            ),
+        )    
+        
+    aligned_images = align_images_to_mean(
+        images          = images,
+        motion_model    = motion_model,
+        feature_method  = feature_method,
+        )
+
+    return aligned_images
+
+
+def alignment_help_widget():
```

Comment:
I'm not sure this needs its own function. To me it makes more sense to put it with the widget that it describes: `align_to_mean_image_widget`.